code/__DEFINES/preferences.dm

```diff
@@ -36,7 +36,7 @@
 #define BE_ALIEN		64
 #define BE_PAI			128
 #define BE_CULTIST		256
-#define BE_NINJA		512
```

Comment:
@Crazylemon64 

I know you were eager to work on organs, but Feaths is workin' on her thing--if you wanted something to do, the antag preference refactor from TG would definitely help us out immensely as we have officially run out of flags.